themes/src/main/resources/theme/keycloak/common/resources/node_modules/rcue/dist/img/git-logo.svg

```diff
@@ -1,21 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we remove this change from this PR? It looks OK, but I would prefer making theme related changes in a separate PR. Otherwise, we are going to need to include a few more people to the review :)

Reply:
That was actually a mistake - I have restored it. Thanks for the catch. 👍